base/interactiveutil.jl

```diff
@@ -2,7 +2,14 @@
 
 # editing files
 
-function edit(file::AbstractString, line::Integer)
+doc"""
+    editor()
+
+Determines the editor to use when running functions like `edit`. Returns an Array compatible
+for use within backticks. You can change the editor by setting JULIA_EDITOR, VISUAL, or
+EDITOR as an environmental variable.
+"""
+function editor()
```

Comment:
`editor` isn't an exported function so I would guess this should not be included in the external documentation. Opinions?